react/week2/my-app/src/App.js

```diff
@@ -0,0 +1,23 @@
+//import logo from './logo.svg';
+import './App.css';
+import ListItems, { todos} from "./List.js";
+import Timer from "./Timer.js"
+
+function Header(props) {
+  return (
+    <h1>{props.title}</h1>
+  )
+}
+
+function App() {
+
+  return (
+    <>
```

Comment:
maybe add some padding to the root div, so the list is not so close to the left edge 🙂